File: Fati/taxgradle/src/main/java/org/tax/parser/Variable.java
package org.tax.parser;

public class Variable {
	String name;
	String value;
	String type;

	public Variable(String name, String value, String type) {
		this.name = name;
		this.value = value;
		this.type = type;
	}
	
	@Override
	public String toString() {
		return this.name +  " : " + this.value + " of type " + this.type; 
	}
	
}

File: guybas/QL/AST/Expressions/Elements/variable.py
import QL.AST.Expressions.Elements.element as e


class Variable(e.Element):
    def __init__(self, name):
        self.name = name

    def return_type_string(self, type_dict):
        return type_dict[self.name]

    def pretty_print(self):
        return self.name

    def get_dependencies(self):
        return [self.name]